Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -52,6 +61,22 @@ size_t saturatedAdd(size_t a, size_t b) {
   res |= -static_cast<int>(res < a);
   return res;
 }
+
+std::string demangle(const char* sym) {
+#if __has_include(<cxxabi.h>)
+  int status = 0;
+
+  char* realname = abi::__cxa_demangle(sym, nullptr, nullptr, &status);
+
+  if (status != 0) {
+    return sym;
+  }
+
+  return realname;
+#else
+  return sym;
+#endif
+}
```

Comment:
should we put this in a central utility header? might be useful in other places

Reply:
I'm removing this entirely in favor of `boost::core::demangle`.